app/views/ems_container/_show_topology.html.haml

```diff
@@ -0,0 +1,84 @@
+.container_topology{'data-ng-app' => "topologyApp", 'ng-controller' => "containerTopologyController"}
```

Comment:
> Right. Unlike dashboard the topology page will be about the same for each entity, ill move this to views/common or the like

in any case (it's not clear to me from your comment when this will be unified/moved to a common location), it should not be merged when there's code duplication.